src/azure-cli-core/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+2.8.0
+++++++
+* Add new get command loader mechanism when loading CommandLoader (#13763)
```

Comment:
```suggestion
* Add new get_command_loader() entry when loading CommandLoader (#13763)
```

Reply:
* Add get_command_loader() entry to support to load customized CommandLoader